sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -610,6 +623,49 @@ public void setSampleRate(Double sampleRate) {
     this.sampleRate = sampleRate;
   }
 
+  /**
+   * Returns the traces sample rate Default is null (disabled)
+   *
+   * @return the sample rate
+   */
+  public @Nullable Double getTracesSampleRate() {
+    return tracesSampleRate;
+  }
+
+  /**
+   * Sets the tracesSampleRate Can be anything between 0.01 and 1.0 or null (default), to disable
+   * it.
+   *
+   * @param sampleRate the sample rate
+   */
+  public void setTracesSampleRate(Double sampleRate) {
```

Comment:
```suggestion
  public void setTracesSampleRate(Double tracesSampleRate) {
```

just to avoid the shadowing with `this.sampleRate`

Reply:
👍